sdk/filter.go

```diff
@@ -0,0 +1,29 @@
+package sdk
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+)
+
+type Filterable interface {
+	FilterSelf(*unstructured.Unstructured) (interface{}, error)
+}
+
+func WrapFilterable(filterable Filterable) go_hook.FilterFunc {
```

Comment:
I think this method and the FilterFunc field are redundant.  Let's rename FilterFunc to Filterable! Also, FilterSelf is not clear. What do you think about `ApplyFilter`? `Apply` prefix is used for `ApplyJqFilter`.

```
// go_hook/go_hook.go

type Filterable interface {
	ApplyFilter(*unstructured.Unstructured) (interface{}, error)
}

type KubernetesConfig struct {
        ...
	Filterable Filterable
}
```

Reply:
Done!